apps/angular/src/core/services/api.service.ts

```diff
@@ -0,0 +1,30 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+
+import { BASE_URL } from '../../constants';
+
+/** Api service methods. */
+@Injectable({
+  providedIn: 'root',
+})
+
+/** Api service methods. */
+export class ApiService {
+
+  public constructor(private readonly httpClient: HttpClient) { }
+
+  /**
+   *  Api service constructors.
```

Comment:
Extra space. Please, check it everywhere
```suggestion
   * Api service constructors.
```